packages/cli/templates/netlify-functions/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "scripts": {
+    "build": "remix build",
+    "predev": "rimraf ./public/_redirects",
+    "dev": "remix dev",
+    "start": "netlify serve",
+    "typecheck": "tsc -b"
+  },
+  "dependencies": {
+    "@netlify/functions": "^1.3.0",
+    "@netlify/remix-adapter": "^1.0.0",
+    "cross-env": "^7.0.3"
```

Comment:
Seem unused

Reply:
yeah, I will clean the netlify templates. Keeping them `1:1` is bringing a lot of un-used stuff too 👍